pyrdp/parser/rdp/ntlmssp.py

```diff
@@ -34,11 +37,11 @@ def findMessage(self, data: bytes) -> int:
         return data.find(b"NTLMSSP\x00")
 
     def doParse(self, data: bytes) -> NTLMSSPPDU:
-        stream = BytesIO(data)
+        sigOffset = self.findMessage(data)
+        stream    = BytesIO(data[sigOffset:])
         signature = stream.read(8)
         messageType = Uint32LE.unpack(stream)
-
-        return self.handlers[messageType](data, stream)
+        return self.handlers[messageType](stream.getvalue(), stream)
```

Comment:
Before, we would pass a ByteIO stream and all data to the handlers.

Now, no matter the `messageType`, we pass a subset of the data starting with `NTLMSSSP\x00` and a partial stream accordingly.

The behavior might be correct to capture NTLM hashes but it is probably wrong for other packet types.

Reply:
This was previously done at `pyrdp/security/nla.py` and moved it here.